extensions/amp-mowplayer/OWNERS.yaml

```diff
@@ -1 +1 @@
-- gopanisandip
+gopanisandip
```

Comment:
`-` should not be removed from this file.

Reply:
Ok this is reverted.  Ideally there should not be any change on this file but due to conflicts previously we have this file got updated.

Anyway, Could you please take a look again in this PR and let me know if any changes required.